doc/recipe/preprocessor.rst

```diff
@@ -798,6 +801,22 @@ between 1 and 12 as the named month string will not be accepted.
 
 See also :func:`esmvalcore.preprocessor.extract_month`.
 
+.. _hourly_statistics:
+
+``hourly_statistics``
+---------------------
+
+This function produces statistics at a x-hourly frequency.
+
+Parameters:
+    * every_n_hours: frequency to use to compute the statistics. Must be a divisor of
+      24.
+
+    * operator: operation to apply. Accepted values are 'mean',
+      'median', 'std_dev', 'min', 'max' and 'sum'. Default is 'mean'
+
+See also :func:`esmvalcore.preprocessor.daily_statistics`.
```

Comment:
this should be:
See also :func:`esmvalcore.preprocessor.hourly_statistics`.
I guess